src/targets/unified.ini

```diff
@@ -146,6 +146,29 @@ extends = env:Unified_ESP32S3_900_TX_via_ETX
 [env:Unified_ESP32S3_900_TX_via_WIFI]
 extends = env:Unified_ESP32S3_900_TX_via_ETX
 
+[env:Unified_ESP32S3_LR1121_TX_via_ETX]
+extends = env_common_esp32s3tx, radio_LR1121
+build_flags =
+	${env_common_esp32s3tx.build_flags}
+	${radio_LR1121.build_flags}
+	${common_env_data.build_flags_tx}
+	-include target/Unified_ESP32_TX.h
+	-D TARGET_AXIS_THOR_2400_TX=1
```

Comment:
This define is no longer required, can you also remove it from the place you found it 👍

Reply:
I pulled those from env:Unified_ESP8285_900_TX_via_UART and since it was working didn't think twice, I took them out of both.